vunit/project.py

```diff
@@ -370,59 +370,74 @@ def _get_compile_timestamps(self, files):
                 timestamps[source_file] = ostools.get_modification_time(hash_file_name)
         return timestamps
 
-    def get_files_in_compile_order(self, incremental=True, dependency_graph=None):
+    def get_files_in_compile_order(self, incremental=True, dependency_graph=None, files=None):
         """
         Get a list of all files in compile order
-        incremental -- Only return files that need recompile if True
+        :param incremental: -- Only return files that need recompile if True
```

Comment:
Is `--` a typo?

Also, please, add descriptions for `dependency_graph` and `files` too.